### DISCUSSION TITLE
Where can I find the change-log?

### DISCUSSION THREAD

**Discussion by Brosten** [xamarin/XamarinCommunityToolkit#1804] [Q&A] [answered]
I see that v 2.0.0 appeared 27:th of jan, but where can I find out what's new in it?

**Comment by bijington** [comment, marked as answer]
You can view it on the Release page here:

https://github.com/xamarin/XamarinCommunityToolkit/releases

**Reply by Brosten** [reply]
Ahh, thanks! Not fully used to guthub yet... :)

**Reply by mkrnic** [reply]
That really doesn't tell much, as most of the stuff is hidden underneath: "Merge Develop to Main by @brminnick in #1790"

Which is probably this, with a whole bunch of things you need to filter out in your head: https://github.com/xamarin/XamarinCommunityToolkit/commit/a3c61c05f9634e18ff1cd4a3e8b6f023c4c9f6fc

**Reply by bijington** [reply]
@mkrnic thanks for pointing this out. Hopefully this should be easier to follow now:

https://github.com/xamarin/XamarinCommunityToolkit/releases/tag/2.0.0
